lazyflow/operators/opPatchCreator.py

```diff
@@ -196,8 +197,8 @@ def execute(self, slot, subindex, roi, result):
             gWidth = self.GridWidth.value
             gHeight = self.GridHeight.value
 
-            numPatchesVertical = int((gHeight - pHeight) / skipVertical) + 1
-            numPatchesHorizontal = int((gWidth - pWidth) / skipHorizontal) + 1
+            numPatchesVertical = (gHeight - pHeight) // skipVertical) + 1
```

Comment:
You are correct. I missed some of the test results last time. Taking care of them along with incorporating other comments. Pls wait for 2 days.